frontend/src/config/navigation.ts

```diff
@@ -27,6 +29,7 @@ export const linking: LinkingOptions<RootStackParamList> = {
       Events: "events",
       Announcements: "announcements",
       ContactUs: "contact-us",
+      AnnouncementDetail: "announcements/:id",
```

Comment:
@hyunghoseo Would this be the right way to setup path for the AnnouncementDetail Page?

Reply:
https://github.com/hyunghoseo/nykea-app/pull/64/files#diff-c51c8dcf322cc8722620e6d22f9b2642059abfcae247a0c9a9b789f3257010f7R57

This was due to linkings not being configured correctly.